File: fabric/src/main/java/com/badmcstudio/frostbitekit/fabric/FrostbiteKitFabric.java
package com.badmcstudio.frostbitekit.fabric;

import com.badmcstudio.frostbitekit.FrostbiteKit;
import net.fabricmc.api.ModInitializer;

public class FrostbiteKitFabric implements ModInitializer {
    @Override
    public void onInitialize() {
        FrostbiteKit.init();
    }
}

File: fabric/src/main/java/com/badmcstudio/frostbitekit/fabric/FrostbiteKitFabric.kt
package com.badmcstudio.frostbitekit.fabric

import com.badmcstudio.frostbitekit.FrostbiteKit
import net.fabricmc.api.ModInitializer


class FrostbiteKitFabric : ModInitializer {
    override fun onInitialize() {
        FrostbiteKit.init()
    }
}